documentation/credscan-process.md

```diff
@@ -0,0 +1,67 @@
+# Guide for monitoring CredScan checks
```

Comment:
Not at all for this PR but I notice we now go back and forth between our GH wiki and our documentation folder...

I almost prefer everything live in the documentation folder as .md files because it's easier to submit and review changes. Should we standardize on one or the other?

Reply:
A good topic to discuss in team meeting?  Yes, committed files make reviewing easier.  Wiki allows quick turnaround that doesn't need a lot of reviews I guess.

Reply:
Good idea, I added to the agenda